src/main/java/com/watch/shop/app/model/service/WatchService.java

```diff
@@ -0,0 +1,17 @@
+package com.watch.shop.app.model.service;
+
+import com.watch.shop.app.model.entity.Watch;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+public interface WatchService {
+
+    List<Watch> getWatches();
+
+    List<Watch> sortWatchesByParam(String param);
+
+    BigDecimal getTotalCost();
+
+    void insertWatch(Watch watch);
```

Comment:
Please use Java naming convention - camel use
void insertWatch(Watch watch)

Reply:
It's a big letter, it looks small to me here too, but in the code it's big
![image](https://github.com/user-attachments/assets/89634d2f-664e-4770-be8c-1dc4f934db32)

Reply:
I see, seems just font issue, too thin strokes. Ok, resolved